src/snovault/attachment.py

```diff
@@ -37,6 +37,13 @@ def includeme(config):
     config.scan(__name__)
 
 
+class InternalRedirect(Response):
+    @staticmethod
+    def _make_location_absolute(environ, value):
+        # Avoid changing Location header so Apache internally redirects.
```

Comment:
It looks like we're overriding the _make_location_absolute for downloads with 'get_blob_url' and 'accel_redirect_header'.  

The override is skipping this line of code in webob, https://github.com/Pylons/webob/blob/master/src/webob/response.py#L1314

What is the difference between returning the value regardless of the condition, 'if SCHEME_RE.searchv(value)' ?

Reply:
```python
    @staticmethod
    def _make_location_absolute(environ, value):
        if SCHEME_RE.search(value):
            return value

        new_location = urlparse.urljoin(_request_uri(environ), value)
        return new_location
```
Looks like the `if SCHEME_RE.searchv(value)` is just a fast path for values which are already absolute urls (so would not be changed by urljoin.)